### PR TITLE
fix: change the brand name to Jina AI

### LINKED ISSUE #8690: Update the brand name from Jina to Jina AI
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.8.3

### Cloud or Self Hosted

Cloud, Self Hosted (Docker), Self Hosted (Source)

### Steps to reproduce

image

image

### ✔️ Expected Behavior

_No response_

### ❌ Actual Behavior

_No response_

### PR DESCRIPTION
**Pull request by hey-sa**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

This PR updates all references to the brand name throughout the codebase from "Jina" to "Jina AI".

fix #8690

Fixes 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by crazywoola — APPROVED** [review]

**Comment by vuisme** [comment]
{7BD64181-D3A3-4A72-88D7-5117DD4F3802}
Missed something

File: api/core/model_runtime/model_providers/jina/jina.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-provider: jina
```
Comment (laipz8200) [inline api/core/model_runtime/model_providers/jina/jina.yaml:1]: Please keep this name the same as the provider's path name, for this provider, it's `jina`.

File: api/core/tools/provider/builtin/jina/jina.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 identity:
   author: Dify
-  name: jina
+  name: Jina AI
```
Comment (laipz8200) [inline api/core/tools/provider/builtin/jina/jina.yaml:3]: Please keep this name the same as the provider's path name, for this provider, it's `jina`.

**Review by laipz8200 — COMMENTED** [review]
~~Hi @Nick17t. It seems some errors were caused by this PR, could you fix them in a new PR and link to this one?~~

Fixed

File: api/core/model_runtime/model_providers/jina/jina.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+provider: Jina AI
```
Comment (laipz8200) [inline api/core/model_runtime/model_providers/jina/jina.yaml:1]: Please keep this name the same as the provider's path name, for this provider, it's jina.

**Pull request merged**